recipes/pybammsolvers/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "pybammsolvers" %}
+{% set version = "0.0.4" %}
+{% set sundials_version = "6.5.0" %}
+{% set suitsparse_version = "6.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pybammsolvers-{{ version }}.tar.gz
+    sha256: 6b76fb61e68af34b3a4803edc8020f508b10195f67857d8010e40f78cbbe2768
+  - url: https://github.com/LLNL/sundials/archive/v{{ sundials_version }}.tar.gz
+    sha256: 7557cdb8fe2a9b56fa0c259aa4e2c2dc488a1c79d482bb7943e27b7d76f6931b
+    folder: sundials
+  - url: https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/refs/tags/v{{ suitsparse_version }}.tar.gz
+    sha256: 7111b505c1207f6f4bd0be9740d0b2897e1146b845d73787df07901b4f5c1fb7
+    folder: suitsparse
+
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.9,<3.13
+    - setuptools >=64
+    - wheel
```

```suggestion
    - setuptools >=71
```

Shouldn't be needed anymore, see https://github.com/pybamm-team/PyBaMM/pull/4652 and https://github.com/pybamm-team/pybammsolvers/pull/14